lib/doctrine.js

```diff
@@ -404,7 +404,7 @@
 
         function TagParser(options, title) {
             this._options = options;
-            this._title = title;
+            this._title = title.toLowerCase();
```

Comment:
Hmmm, I'm not sure this is the right thing to do. Swallowing letter casing isn't really what parsers do.